.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+    push:
+        branches: [ master, '[1-9]+.x' ]
+    pull_request:
+
+jobs:
+    tests:
+        name: Tests
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php: [ '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0' ]
+
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    coverage: "none"
+                    php-version: "${{ matrix.php }}"
+
+            -   name: Install dependencies
+                run: composer update --ansi --no-progress
```

Comment:
The Github Actions environment is detected as an environment not using colors by symfony/console, hence the `--ansi` option to force colors (`--colors=always` does the same for PHPUnit below as its detection logic for `--colors=auto` is basically the same). `--no-progress` avoids rendering the download progress bar, because the rendering of logs of github actions does not support the overriding of lines, and so shows all successive displays of the progress bar, which is confusing.

Compared to our Travis config, I'm using `composer update` because Composer 2 complains when using `install` without a lock file (which is turned into an `update` internally).